src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs

```diff
@@ -75,10 +84,13 @@ private void EnsureStandardPopupAlignment()
 
         private void PopupWindow_Closed(object sender, EventArgs e)
         {
-            if (webBrowserWindow != null)
-                webBrowserWindow.IsOpen = false;
+            if (webBrowserComponent != null)
+            {
+                webBrowserComponent.Visibility = Visibility.Collapsed;
```

Comment:
@RobertGlobant20 Can you explain when is the `webBrowserComponent` disposed?

Reply:
@QilongTang the WebView2 control is created every time the Popup is opened and disposed when the Popup is Hidden (specifically in the Step.Hide() method that is executed when we click the next or prev Popup button), the problem with Popup.IsOpen = false is that it generates an async call to Popup.OnClose() so if we don't dispose the WebView2 instance before we close the PackageManagerSearch, the WebView2 crashes (in the image below you will see that we set IsOpen = false and then we execute the UIAutomation Steps - inside one of this we are closing PackageManagerSearch). 
![image](https://user-images.githubusercontent.com/61755417/182259722-e8aa8b15-6f4e-4157-a499-8430c262d8f3.png)

Reply:
@RobertGlobant20 Gotcha, thanks!